ApollosProject/apollos-prototype
# Adds Data Connector Analytics

## DESCRIPTION

### What does this PR do, or why is it needed?

Splits Analytics into it's own package. Works exactly the same as before.

### What design trade-offs/decisions were made?

Chose to delay putting all the testing related components into their own package.

### How do I test this PR?

Boot up app. Browse around. Visit a content item. Ensure you don't see any errors in the api console. 

### What automated tests did you write?

n/a - moved existing tests.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
